_posts/2025-03-20-phased-strimzi-upgrade-example.md

```diff
@@ -0,0 +1,637 @@
+---
+layout: post
+title: "Phased upgrades of Strimzi managed Kafka fleets"
+date: 2025-03-20
+author: tom_cooper
+---
+
+## TL;DR
```

Comment:
I would remove this title, it seems silly and unnecessary. All it does is it provides a simple introduction as all out other blog posts.